### ISSUE TITLE
Tests fail

### ISSUE THREAD

**Issue by zoffixznet** [niner/Inline-Perl5#46]
Tried to install on latest (Oct 8, 2015) rakudo, but tests fail:
```bash
zoffix@ZofMain:/tmp$ panda Inline::Perl5
==> Inline::Perl5 depends on LibraryMake, File::Temp
==> File::Temp depends on File::Directory::Tree
==> Fetching LibraryMake
==> Building LibraryMake
==> Testing LibraryMake
t/01-basic.t .. ok
All tests successful.
Files=1, Tests=7,  1 wallclock secs ( 0.02 usr  0.00 sys +  1.05 cusr  0.06 csys =  1.13 CPU)
Result: PASS
==> Installing LibraryMake
Copying blib/lib/LibraryMake.pm6 to /home/zoffix/.rakudobrew/moar-nom/install/share/perl6/site/lib/LibraryMake.pm6
==> Successfully installed LibraryMake
==> Fetching File::Directory::Tree
==> Building File::Directory::Tree
==> Testing File::Directory::Tree
t/basic.t .. ok
All tests successful.
Files=1, Tests=7,  1 wallclock secs ( 0.01 usr  0.01 sys +  0.35 cusr  0.03 csys =  0.40 CPU)
Result: PASS
==> Installing File::Directory::Tree
Copying blib/lib/File/Directory/Tree.pm to /home/zoffix/.rakudobrew/moar-nom/install/share/perl6/site/lib/File/Directory/Tree.pm
==> Successfully installed File::Directory::Tree
==> Fetching File::Temp
==> Building File::Temp
==> Testing File::Temp
t/tempfile.t .. ok
All tests successful.
Files=1, Tests=14,  1 wallclock secs ( 0.01 usr  0.00 sys +  0.62 cusr  0.02 csys =  0.65 CPU)
Result: PASS
==> Installing File::Temp
Copying blib/lib/File/Temp.pm to /home/zoffix/.rakudobrew/moar-nom/install/share/perl6/site/lib/File/Temp.pm
==> Successfully installed File::Temp
==> Fetching Inline::Perl5
==> Building Inline::Perl5
gcc -Wall p5helper.c `perl -MExtUtils::Embed -e ccopts -e ldopts` -shared -o /tmp/.panda-work/1444353959_4/blib/lib/Inline/p5helper.so -fPIC -g
p5helper.c: In function ‘p5_hv_store’:
p5helper.c:212:5: warning: value computed is not used [-Wunused-value]
==> Testing Inline::Perl5
t/autoload.t .................. ok
t/call.t ...................... ok
t/call_back.t ................. ok
t/call_hash.t ................. ok
t/call_p6_function_from_p5.t .. ok
t/callables.t ................. ok
t/can.t ....................... ok
t/eval.t ...................... ok
t/eval_p6_in_p5.t ............. ok
t/eval_return_values.t ........ ok
t/exceptions.t ................ ok
t/file_handles.t .............. ok
t/from.t ...................... ok
t/import.t .................... ok
t/inherit.t ................... ok
t/invoke.t .................... ok
t/invoke_p6_method_from_p5.t .. ok

# Failed test at t/overload.t line 35
#      expected: '/Perl5Object\<\d+\>/'
#      got: 'Perl5Object<-1226542272>'
# Looks like you failed 1 test of 4
t/overload.t .................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests 
t/p5_to_p6.t .................. ok
t/p6_object_destructor.t ...... ok
t/p6_to_p5.t .................. ok
t/perl5parent.t ............... ok
t/precomp.t ................... ok

# Failed test at t/stringify.t line 25
#      expected: '/Bar\<\d+\>/'
#      got: 'Bar<-1226652496>'
# Looks like you failed 1 test of 2
t/stringify.t ................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests 
t/use.t ....................... 
All 4 subtests passed 
t/v6.t ........................ ok

Test Summary Report
-------------------
t/overload.t                (Wstat: 256 Tests: 4 Failed: 1)
  Failed test:  4
  Non-zero exit status: 1
t/stringify.t               (Wstat: 256 Tests: 2 Failed: 1)
  Failed test:  2
test stage failed for Inline::Perl5: Tests failed
  in method throw at /home/zoffix/.rakudobrew/moar-nom/install/share/perl6/runtime/CORE.setting.moarvm:1
  in method install at /home/zoffix/.rakudobrew/moar-nom/install/share/perl6/site/lib/Panda.pm:142
  in method resolve at /home/zoffix/.rakudobrew/moar-nom/install/share/perl6/site/lib/Panda.pm:219
  in sub MAIN at /home/zoffix/.rakudobrew/bin/../moar-nom/install/share/perl6/site/bin/panda:18
  in sub MAIN at /home/zoffix/.rakudobrew/bin/../moar-nom/install/share/perl6/site/bin/panda:91
  in block <unit> at /home/zoffix/.rakudobrew/bin/../moar-nom/install/share/perl6/site/bin/panda:95


Failure Summary
----------------
Inline::Perl5(
	*test stage failed for Inline::Perl5: Tests failed)
zoffix@ZofMain:/tmp$ 
```

**Comment by LLFourn** [comment]
also failing for me on OSX with latest rakudo:

```
bash-3.2$ panda install Inline::Perl5
==> Fetching Inline::Perl5
==> Building Inline::Perl5
gcc -Wall p5helper.c `perl -MExtUtils::Embed -e ccopts -e ldopts` -shared -o /Users/llfourn/src/GTF/.panda-work/1445438779_1/blib/lib/Inline/p5helper.dylib -fPIC -g
==> Testing Inline::Perl5
t/autoload.t .................. ok
t/call.t ...................... ok
t/call_back.t ................. ok
t/call_hash.t ................. ok
t/call_p6_function_from_p5.t .. ok
t/callables.t ................. ok
t/can.t ....................... ok
t/eval.t ...................... ok
t/eval_p6_in_p5.t ............. ok
t/eval_return_values.t ........ ok
t/exceptions.t ................ ok
t/file_handles.t .............. ok
t/from.t ...................... ok
t/import.t .................... ok
t/inherit.t ................... ok
t/invoke.t .................... ok
t/invoke_p6_method_from_p5.t .. ok
t/overload.t .................. ok
t/p5_to_p6.t .................. ok
t/p6_object_destructor.t ...... ok
t/p6_to_p5.t .................. ok
t/perl5parent.t ............... ok
t/precomp.t ................... ok
t/stringify.t ................. ok
t/use.t ....................... 
All 4 subtests passed 
t/v6.t ........................ ok

Test Summary Report
-------------------
t/use.t                     (Wstat: 11 Tests: 4 Failed: 0)
  Non-zero wait status: 11
Files=26, Tests=345, 108 wallclock secs ( 0.08 usr  0.04 sys + 97.25 cusr  3.23 csys = 100.60 CPU)
Result: FAIL
test stage failed for Inline::Perl5: Tests failed
  in method throw at /Users/llfourn/.rakudobrew/moar-nom/install/share/perl6/runtime/CORE.setting.moarvm:1
  in method install at /Users/llfourn/.rakudobrew/moar-nom/install/share/perl6/site/lib/Panda.pm:142
  in method resolve at /Users/llfourn/.rakudobrew/moar-nom/install/share/perl6/site/lib/Panda.pm:219
  in sub MAIN at /Users/llfourn/.rakudobrew/bin/../moar-nom/install/share/perl6/site/bin/panda:18
  in block <unit> at /Users/llfourn/.rakudobrew/bin/../moar-nom/install/share/perl6/site/bin/panda:145


Failure Summary
----------------
Inline::Perl5(
	*test stage failed for Inline::Perl5: Tests failed)
```

**Comment by 0racle** [comment]
Same tests as OP fail for me. Running Ubuntu 15.10 and latest Perl/MoarVM/NQP built from source (as of 00:30 03/11/2015).

    # Failed test at t/overload.t line 35
    #      expected: '/Perl5Object\<\d+\>/'
    #      got: 'Perl5Object<-1226988604>'
    # Looks like you failed 1 test of 4
    t/overload.t .................. 
    Dubious, test returned 1 (wstat 256, 0x100)
    Failed 1/4 subtests 
    
    # Failed test at t/stringify.t line 25
    #      expected: '/Bar\<\d+\>/'#      expected: '/Bar\<\d+\>/'#      got: 'Bar<-1227563412>'
    # Looks like you failed 1 test of 2
    t/stringify.t ................. 
    Dubious, test returned 1 (wstat 256, 0x100)
    Failed 1/2 subtests 
    
    Test Summary Report
    -------------------
    t/overload.t                (Wstat: 256 Tests: 4 Failed: 1)
      Failed test:  4
      Non-zero exit status: 1
    t/stringify.t               (Wstat: 256 Tests: 2 Failed: 1)
      Failed test:  2
      Non-zero exit status: 1
    Files=26, Tests=345, 160 wallclock secs ( 0.12 usr  0.03 sys + 155.57 cusr  2.67 csys = 158.39 CPU)
    Result: FAIL
    test stage failed for Inline::Perl5: Tests failed
      in method throw at /opt/perl6/share/perl6/runtime/CORE.setting.moarvm:1
      in method install at /opt/perl6/share/perl6/site/lib/Panda.pm:142
      in method resolve at /opt/perl6/share/perl6/site/lib/Panda.pm:219
      in sub MAIN at /opt/perl6/share/perl6/site/bin/panda:18
      in block <unit> at /opt/perl6/share/perl6/site/bin/panda:145

**Comment by niner** [comment]
Sorry for the delay. Fixed in commit ffeb71abf4922a9b59994a2eec9305de6fe58bad
